tests/test_auth.py

```diff
@@ -121,8 +130,12 @@ def test_auth_jwt_expiry(client, db_name, root_password, secret):
     db.conn._auth_header = f"bearer {expired_token}"
     assert isinstance(db.version(), str)
 
-    # Test correct error on token expiry.
+    # Test correct error on token expiry (superuser).
     db = client.db("_system", superuser_token=expired_token)
     with assert_raises(ServerVersionError) as err:
         db.version()
     assert err.value.error_code == FORBIDDEN
+
+    # Test correct error on token expiry (user).
+    with assert_raises(JWTExpiredError) as err:
+        db = client.db("_system", user_token=expired_token)
```

Comment:
Could add an additional test for `set_token` here?

Reply:
good catch! added in [74e1b0b](https://github.com/ArangoDB-Community/python-arango/pull/293/commits/74e1b0bb4b6f826577094d9fb5b0cf186cfb4b86)